src/assignment4/dbConnection/DBConnect.java

```diff
@@ -0,0 +1,37 @@
+package assignment4.dbConnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.util.Properties;
+
+public class DBConnect {
+
+  private static DBConnect singleInstance;
+  private Connection connection;
+  private final static String driver = "com.mysql.cj.jdbc.Driver";
+  private final static String url = "jdbc:mysql://127.0.0.1:3306/assignment";
+
+  private DBConnect() throws Exception {
+    Class.forName(driver);
+    Properties info = new Properties();
+    info.put("user", "admin");
```

Comment:
move username and password to config. create .yaml file and read credentials from there.